app/src/main/java/org/simple/clinic/illustration/HomescreenIllustrationRepository.kt

```diff
@@ -20,13 +21,14 @@ class HomescreenIllustrationRepository @Inject constructor(
     private val userClock: UserClock
 ) {
 
-  fun illustrations(): Observable<GetFileResult.Success> =
+  fun illustrations(): Observable<File> =
```

Comment:
This name seems inconsistent with the return type since it indicates that it would return a `Collection` of some sort, but instead it returns a single `File`.

I will rename this to something more _obvious_, like `illustrationImageToShow`. What do you think?